examples/udp_server.rs

```diff
@@ -1,58 +1,75 @@
+use std::io;
+
+use log::warn;
 use mio::net::UdpSocket;
 use mio::{Events, Interests, Poll, Token};
-use std::io;
 
-const IN: Token = Token(0);
+// A token to allow us to identify which event is for the `UdpSocket`.
+const UDP_SOCKET: Token = Token(0);
 
 fn main() -> io::Result<()> {
-    // Set up a new socket on port 9000 to listen on.
-    let socket = UdpSocket::bind("0.0.0.0:9000".parse().unwrap())?;
-    // Initialize poller.
+    env_logger::init();
+
+    // Create a poll instance.
     let mut poll = Poll::new()?;
-    // Register our socket with the token IN (defined above) and an interest
-    // in being `READABLE`.
-    poll.registry().register(&socket, IN, Interests::READABLE)?;
-
-    // Prepare a buffer for the number of events we can handle at a time.
-    // Someone might wat to echo really fast so lets give it some size.
-    let mut events = Events::with_capacity(1024);
-    // Initialize a buffer for the UDP datagram
-    let mut buf = [0; 65535];
-    // Main loop
+    // Create storage for events, since we will only register a single socket a
```

Comment:
TIOLI: "Create storage for events. Since we will only register a single socket, a capacity of 1 will do."